docs/src/main/sphinx/release/release-423.md

```diff
@@ -5,6 +5,7 @@
 * Add support for renaming nested fields in a column via `RENAME COLUMN`. ({issue}`16757`)
 * Add support for setting the type of a nested field in a column via `SET DATA TYPE`. ({issue}`16959`)
 * Add support for comments on materialized view columns. ({issue}`18016`)
+* Show original characters for string literals. ({issue}`5061`)
```

Comment:
I'm not really sure what this means.

Reply:
Yeah .. not sure how to reword and also where it shows up .. the PR talk about displaying unicode symbols used for literals when doing a SHOW CREATE TABLE .. but I have no idea how wide the scope is .. for example I think it will also apply for SHOW CREATE SCHEMA or SHOW CREATE VIEW and so on..